src/server/alternances/infra/repositories/apiLaBonneAlternance.mapper.ts

```diff
@@ -155,8 +155,8 @@ function mapLbaCompanies(lbaCompanies: AlternanceApiJobsResponse['lbaCompanies']
 }
 
 export const mapAlternanceListe = (response: AlternanceApiJobsResponse): ResultatRechercheAlternance => {
-	const matchas = response.matchas.results.map(mapRésultatRechercherAlternanceMatcha);
-	const peJobs = response.peJobs.results.map(mapRésultatRechercherAlternancePEJob);
+	const matchas = !response.matchas ? [] : response.matchas.results.map(mapRésultatRechercherAlternanceMatcha);
+	const peJobs = !response.peJobs ? [] : response.peJobs?.results.map(mapRésultatRechercherAlternancePEJob);
```

Comment:
nitpick : je trouve un chouilla plus lisible comme ça 🤓

```suggestion
	const matchas = response.matchas?.results.map(mapRésultatRechercherAlternanceMatcha) ?? [];
	const peJobs = response.peJobs?.results.map(mapRésultatRechercherAlternancePEJob) ?? [];
```